### PR TITLE
Add aspects to curriculum/javascript/core/arrays/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/arrays/arrays.md?ref=b4aa97803e928f1c5ba2f5ae6cdbbd20c62a8131

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

File: javascript/core/arrays/js-practice-glitch-array-for-loop.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ levels:
 tags:
 
   - introduction
+
 aspects:
-  - introduction
+  - workouts
```
Comment (ryanwarsaw) [inline javascript/core/arrays/js-practice-glitch-array-for-loop.md]: Should be `workout`, that's why this is failing. You have an extra `s`.

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Comment by enki-hq** [comment]
Rule insight:aspects failed for files: 
  - javascript/core/arrays/js-practice-glitch-array-for-loop.md

**Pull request merged**